doc/source/drivers/vector/miramon.rst

```diff
@@ -0,0 +1,374 @@
+.. _vector.miramon:
+
+MiraMon Vector
+====================
+
+.. shortname:: MiraMonVector
+
+.. built_in_by_default::
+
+This driver is capable of translating (reading and writing) structured vectors
+of point, arc (*linestrings*), and polygon types from MiraMon vector format.
+
+In MiraMon the concepts of OGRMultiPoints and OGRMultiLineStrings are not supported,
+but the driver translates a multipoint into N points and a multistring into N arcs.
+So, when reading a MiraMon file of type *.pol*, the corresponding
```

Comment:
```suggestion
When reading a MiraMon file of type *.pol*, the corresponding
```

Reply:
I've removed the "So, " which indeed a bit confusing

Reply:
Thanks Even.